src/rendered-deployment.yaml

```diff
@@ -225,7 +225,7 @@ spec:
             - name: OtlpExportTraces
               value: "false"
             - name: ApplicationVersion
-              value: 0.2.244
+              value: 0.2.246
             - name: ChartVersion
               value: 1.0.132
```

Comment:
do we need to update chart version here?

Reply:
No
We need to clean up the deployment and remove this env var and more probably